pkg/scheduler/api/cluster_info.go

```diff
@@ -24,15 +24,16 @@ import (
 
 // ClusterInfo is a snapshot of cluster by cache.
 type ClusterInfo struct {
-	Jobs                 map[JobID]*JobInfo
-	Nodes                map[string]*NodeInfo
-	HyperNodesListByTier map[int][]string
-	HyperNodes           map[string]sets.Set[string]
-	Queues               map[QueueID]*QueueInfo
-	NamespaceInfo        map[NamespaceName]*NamespaceInfo
-	RevocableNodes       map[string]*NodeInfo
-	NodeList             []string
-	CSINodesStatus       map[string]*CSINodeStatusInfo
+	Jobs                      map[JobID]*JobInfo
+	Nodes                     map[string]*NodeInfo
+	HyperNodesListByTier      map[int]sets.Set[string]
+	RealNodesSet              map[string]sets.Set[string]
```

Comment:
Do we need to unify the variable called `HyperNodes` in Session struct to RealNodesSet?

Reply:
done